### PR TITLE
feat: registry page

### PR DESCRIPTION
**Pull request by simboonlong**
### pr updates
- registry page redesign to cater to multiple stores
- refactored to Hero component for use in collaboration + registry pages
- add search
- alignment fixes

### preview
Screenshot 2020-08-31 at 1 00 06 PM

### CODE REVIEW

File: src/components/UI/RegistryCard/RegistryCard.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                            <div>
+                              <a href={`tel:${info.phone}`}>{info.phone}</a>
+                            </div>
+                          )}
+                          {info.description && <div>{info.description}</div>}
+                          {info.address && (
+                            <div className="mt-2">
+                              <div className="text-uppercase">Certificate Store:</div>
+                              <a
+                                href={`https:etherscan.io/address/${info.address}`}
```
Comment (gjj) [inline src/components/UI/RegistryCard/RegistryCard.tsx:76]: Think this should be `https://etherscan.io/address/${info.address}`

Currently, the link points to https://deploy-preview-550--opencerts-development.netlify.app/etherscan.io/address/0x007d40224f6562461633ccfbaffd359ebb2fc9ba

**Review by gjj — APPROVED** [review]
lgtm!

File: src/components/UI/Search/Search.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  onSearchSubmit,
+}: SearchProps) => {
+  return (
+    <form className={`container-fluid ${css.search}`} onSubmit={onSearchSubmit}>
+      <div className="row h-100">
+        <input
+          className="col-9 col-sm-10"
+          data-testid="registry-search"
+          type="search"
+          placeholder="Certificate store"
```
Comment (Nebulis) [inline src/components/UI/Search/Search.tsx:23]: Search certificate store address (as in the story)

File: src/components/UI/RegistryCard/RegistryCard.tsx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    name: string;
+    address?: string;
+    logo: string;
+    email?: string;
+    phone?: string;
+    description: string;
+  }[];
+}
+
+const searchThreshold = 2;
```
Comment (Nebulis) [inline src/components/UI/RegistryCard/RegistryCard.tsx:20]: maybe a small comment why 2 ? ( I guess it's for 0x)

Comment (simboonlong) [reply src/components/UI/RegistryCard/RegistryCard.tsx:20]: yeah, yup okay

**Review by Nebulis — DISMISSED** [review]

**Review by Nebulis — APPROVED** [review]

**Pull request merged**